sqllogictest/src/parser.rs

```diff
@@ -784,21 +814,24 @@ fn parse_inner<T: ColumnType>(loc: &Location, script: &str) -> Result<Vec<Record
                             .try_collect()?;
                         let sort_mode = res
                             .first()
-                            .map(|&s| SortMode::try_from_str(s))
-                            .transpose()
-                            .map_err(|e| e.at(loc.clone()))?;
-                        let label = res.get(1).map(|s| s.to_string());
-                        QueryExpect::Results {
-                            types,
-                            sort_mode,
-                            result_mode: None,
-                            label,
-                            results: Vec::new(),
-                        }
+                            .and_then(|&s| SortMode::try_from_str(s).ok()); // Ignore sort mode if parsing failed
```

Comment:
Is it because it could also be a token for `RetryConfig`?

Reply:
Exactly.